package.json

```diff
@@ -29,10 +29,11 @@
       }
     ],
     "iosInstallScript": "./scripts/install_ios.sh",
-    "androidInstallScript": "./scripts/install_android.sh"
+    "androidInstallScript": "./scripts/install_android.sh",
+    "skipDependencyCheck": true
```

Comment:
An explanation of why we need this for `ads` in particular would be awesome. I'm sure others will wonder why we're skipping the dependencies for it in the future.

Reply:
We never merged in the error handling code, that's from that PR. I'll merge it in now.

For context though (I actually talked this through with you a couple weeks ago) currently packager will always automatically add as direct dependencies any library that's seen as "native", i.e. has native code. When we want admob to silently fail, we don't want this to happen. So, I manually add the dependencies required in the install scripts and tell packager to skip the dependency check for this one.

Reply:
@scott-massey right, I remember that. I guess what I'm asking for is a reference to documentation explaining that because I know someone will be asking about that in the future and won't know why it's being done. For example, it took me looking through the PR to remember that it was related to something else and didn't really have to do with the other changes happening in the first place.

Reply:
What I'm trying to say is, do we have somewhere in Notion/Codebase explaining why this is done the way it is? As long as we have it documented somewhere where an engineer can easily find it, I think we're good to go.